Snowflake.Data.Tests/IntegrationTests/SFConnectionIT.cs

```diff
@@ -46,6 +47,7 @@ public void TestBasicConnection()
 
                 conn.Close();
                 Assert.AreEqual(ConnectionState.Closed, conn.State);
+                SnowflakeDbConnectionPool.SetPooling(pooling);
```

Comment:
here you clean up the changes in pooling, but in the subsequent tests you don't...

Reply:
I was fixing just necessary things as we encountered flaky tests